lib/runtime/wasmer/instance.go

```diff
@@ -94,6 +97,12 @@ func NewInstance(code []byte, cfg *Config) (*Instance, error) {
 		return nil, errors.New("code is empty")
 	}
 
+	var err error
```

Comment:
nit it would be nice to define `err` as a named return since we need it now

```go
func NewInstance(code []byte, cfg *Config) (instance *Instance, err error) {
```